testsuite/MDAnalysisTests/coordinates/test_h5md.py

```diff
@@ -493,9 +358,325 @@ def test_no_units(h5md_file, ref, tmpdir):
+                           convert_units, units):
+        with Writer(outfile,
+                    universe.atoms.n_atoms,
+                    velocities=True,
+                    forces=True,
+                    convert_units=convert_units) as W:
+            for ts in universe.trajectory:
+                W.write(universe)
+
+        uw = mda.Universe(TPR_xvf, outfile)
+        uw_units = uw.trajectory.units.values()
+        for u1, u2 in zip(units, uw_units):
+            assert_equal(u1, u2)
+
+    #@pytest.mark.parametrize('chunks', (None, (1, 1000, 3)))
+    def test_write_chunks(self, universe, outfile, Writer):
+        chunks = (1, 1000, 3)
+        with Writer(outfile,
+                    universe.atoms.n_atoms,
+                    chunks=chunks) as W:
+            for ts in universe.trajectory:
+                W.write(universe)
+
+        uw = mda.Universe(TPR_xvf, outfile)
+        dset = uw.trajectory._particle_group['position/value']
+        assert_equal(dset.chunks, chunks)
+
+    @pytest.mark.parametrize('filter, opts', (('gzip', 1),
+                                              ('gzip', 9),
+                                              ('lzf', None)))
+    def test_write_with_compression(self, universe,
+                                    outfile, Writer,
+                                    filter, opts):
+        with Writer(outfile,
+                    universe.atoms.n_atoms,
+                    compression=filter,
+                    compression_opts=opts) as W:
+            for ts in universe.trajectory:
+                W.write(universe)
+
+        uw = mda.Universe(TPR_xvf, outfile)
+        dset = uw.trajectory._particle_group['position/value']
+        assert_equal(dset.compression, filter)
+        assert_equal(dset.compression_opts, opts)
+
+    @pytest.mark.parametrize('driver, comm', (('core', None),
+                                              ('mpio', 'comm')))
+    def test_write_with_drivers(self, universe,
+                                outfile, Writer,
+                                driver, comm):
+        try:
+            with Writer(outfile,
+                        universe.atoms.n_atoms,
+                        driver=driver,
+                        comm=comm) as W:
+                for ts in universe.trajectory:
+                    W.write(universe)
+
+            uw = mda.Universe(TPR_xvf, outfile, driver=driver, comm=comm)
+            file = uw.trajectory._file
+            assert_equal(file.driver, driver)
+        except:
+            with pytest.raises(ValueError):
+                with Writer(outfile,
+                            universe.atoms.n_atoms,
+                            driver=driver,
+                            comm=comm) as W:
+                    for ts in universe.trajectory:
+                        W.write(universe)
+
+    def test_timestep_not_modified_by_writer(self, universe, Writer, outfile):
```

Comment:
Is this a concern?''

I am generally all in favor of testing everything but if there's no possibility that this happens, I'd remove it to make the tests pass a shred faster.

Reply:
As discussed in our meeting, leave it as it seems to be useful and done for other writers.